data/aerial-photography/naip/index.html

```diff
@@ -19,12 +19,17 @@
   <figcaption class="caption__text">NAIP Sample</figcaption>
 </figure>
 {%- capture abstract -%}
-National Agricultural Imagery Program</a> is color aerial photography typically collected every 2 or 3 years in the
+The U.S. Department of Agriculture <a href="https://www.usgs.gov/centers/eros/science/usgs-eros-archive-aerial-photography-national-agriculture-imagery-program-naip">
+National Agricultural Imagery Program</a> (NAIP) is aerial photography typically collected every 2 or 3 years in the
 middle of the summer. Starting in 2006 the acquisitions have been statewide. The resolution is usually 1 meter with a
 horizontal positional accuracy of -/+ 5 meters. The 2018 & 2021 imagery is .6 meter (60 centimeter) resolution with a horizontal
-positional accuracy of -/+ 4 meters. For more information about the history of the NAIP collections take a look at this 
+positional accuracy of -/+ 4 meters.
+</p>
+<p>For more information about the history of the NAIP collections take a look at this 
 <a href='https://www.arcgis.com/apps/MapTour/index.html?appid=445e3dfd16c4401f95f78ad5905a4cce&webmap=6d117f2ede424ace93917e2bc88202b3'>
-  Map Tour from the USDA</a> or visit their <a href='https://naip-usdaonline.hub.arcgis.com/'>NAIP GeoHub</a>.
+Map Tour from the USDA</a> or visit their <a href='https://naip-usdaonline.hub.arcgis.com/'>NAIP GeoHub</a>. A 4-band imagery service of the
+most current NAIP imagery for all the states in the U.S. is also available from USDA. The USDA NAIP GIS Service URL is 
+https://gis.apfo.usda.gov/arcgis/rest/services/NAIP.</p>
```

Comment:
```suggestion
<a href="https://gis.apfo.usda.gov/arcgis/rest/services/NAIP">https://gis.apfo.usda.gov/arcgis/rest/services/NAIP</a>.</p>
```
It might be nice to make this a link